sql/updates/world/2016_02_HEADLESS_HORSEMAN_world.sql

```diff
@@ -0,0 +1,5 @@
+-- add emote text for Headless Horseman NPC 23682 + Head of the Horseman 23775
+DELETE FROM `creature_text` WHERE `entry` IN (23682,23775) AND `BroadcastTextId`= 18646;
+INSERT INTO `creature_text` (`entry`,`groupid`,`id`,`text`,`type`,`language`,`probability`,`emote`,`duration`,`sound`,`BroadcastTextId`,`TextRange`,`comment`) VALUES
+(23682,5,0,'%s laughs.',16,0,100,0,0,11965,18646,1,'Headless Horseman, EMOTE_HORSEMAN_LAUGHS'),
+(23775,2,0,'%s laughs.',16,0,100,0,0,11965,18646,1,'Head of the Horseman, EMOTE_HEAD_LAUGHS');
```

Comment:
OK, I just went by the original script, but I presume that one was wrong too.
The PR will become even more simple, removal of 3 lines. :)